document/js-api/index.bs

```diff
@@ -443,12 +443,11 @@ The verification of WebAssembly type requirements is deferred to the
             1. Let [=external value|table=] |tableaddr| be |externval|.
             1. Let |table| be [=create a Table object|a new Table object=] created from |tableaddr|.
             1. Let |value| be |table|.
-        1. If |externtype| is of the form [=externtype/tag=] |attribute| <var ignore>functype</var>,
-            1. Assert: |attribute| is [=tagtype/attribute/exception=].
```

Comment:
Why remove this? (Do we still have the attribute thing? If no, needs some more changes.)

Reply:
Oh we do still have the attribute thing, but I was just wondering if "attribute tagtype" made sense. Didn't really mean to remove the assertion though. How can we fix this?
```
        1. If |externtype| is of the form [=externtype/tag=] <var ignore>tagtype</var>,
            1. Assert: |externtype|.|attribute| is [=tagtype/attribute/exception=].
```
Would this be better? Does `|externtype|.|attribute|` make sense?

Reply:
Not really - the existing prose is the exact same pattern as the

> If *externtype* is of the form global *mut* *globaltype*,

step above - binding two new variables with the values of the *externtype* variant. This is what Rust would express as
```rust
match externtype {
  Func(_functype) => ...,
  Global(_mut, _globaltype_) => ...,
  Tag(attribute, _tagtype) => ...,
}
```

Reply:
I see, thanks. I'll revert this.